Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.yml

```diff
@@ -80,42 +85,244 @@ script:
+    - contextPath: File.SHA256
+      description: SHA256 of the file.
+      type: String
+    - contextPath: File.SHA512
+      description: SHA512 of the file.
+      type: String
+    - contextPath: LIS.File.Categories
+      description: Categories matching this file.
+      type: String
+    - contextPath: LIS.File.Risk
+      description: Risk associated to this file.
+      type: String
+    - contextPath: LIS.File.TLP
+      description: TLP level.
+      type: String
+    - contextPath: LIS.File.UsageMode
+      description: Usage mode for LIS.
+      type: String
+    - contextPath: LIS.File.Value
+      description: Hash of the file.
+      type: String
+    - contextPath: LIS.File.Vulnerabilities
+      description: Vulnerabilities associated to this file.
+      type: String
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: String
+  - name: domain
+    description: search IOCs for domains in Gatewatcher's CTI database.
+    arguments:
+    - name: domain
+      description: list of domains to search for, (comma separated values).
+      required: true
+      isArray: true
+    outputs:
+    - contextPath: DBotScore.Indicator
+      description: The indicator that was tested.
+      type: String
+    - contextPath: DBotScore.Type
+      description: The indicator type.
+      type: String
+    - contextPath: DBotScore.Vendor
+      description: The vendor used to calculate the score.
+      type: String
+    - contextPath: DBotScore.Score
+      description: The actual score.
+      type: Number
+    - contextPath: DBotScore.Reliability
+      description: Reliability of the source providing the intelligence data.
+      type: String
+    - contextPath: Domain.Name
+      description: Domain found.
+      type: String
+    - contextPath: Domain.Description
+      description: description of the domain.
+      type: String
+    - contextPath: Domain.TrafficLightProtocol
+      description: TLP level.
+      type: String
+    - contextPath: LIS.Domain.Categories
+      description: Categories matching this domain.
+      type: String
+    - contextPath: LIS.Domain.Risk
+      description: Risk associated to this domain.
+      type: String
+    - contextPath: LIS.Domain.TLP
+      description: TLP level.
+      type: String
+    - contextPath: LIS.Domain.Type
+      description: Type of domain.
+      type: String
+    - contextPath: LIS.Domain.UsageMode
+      description: Usage mode for LIS.
+      type: String
+    - contextPath: LIS.Domain.Value
+      description: The domain name.
+      type: String
+    - contextPath: LIS.Domain.Vulnerabilities
+      description: Vulnerabilities associated to this domain.
       type: String
-  dockerimage: demisto/python3:3.10.12.63474
+  dockerimage: demisto/python3:3.10.13.75921
   runonce: false
   script: '-'
   subtype: python3
   type: python
+  feed: false
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
```

Comment:
I assume this section was added automatically how did you update the yml file?

Reply:
these lines were generated by demisto-sdk, i believe by the command `demisto-sdk format -ud -nv -i Packs/Gatewatcher-LIS/Integrations/LastInfoSec/LastInfoSec.yml`, however i cannot reproduce their generation by removing then and re running the command (demisto-sdk 1.20.7)